rules/windows/lateral_movement_powershell_remoting_target.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2020/11/24"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/24"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies remote execution via PowerShell Remoting on the target host. This could be an indication of lateral movement."
```

Comment:
```suggestion
description = "Identifies remote execution via Windows PowerShell remoting. Windows PowerShell remoting allows for running any Windows PowerShell command on one or more remote computers. This could be an indication of lateral movement."
```